command/job_dispatch.go

```diff
@@ -32,6 +32,12 @@ General Options:
 
 Dispatch Options:
 
+  -meta <key>=<value>
+    Meta takes a key/value pair seperated by "=". The metadata key will be
+    injected into the job's metadata. The flag can be provided more than
+    once to inject multiple metadata key/value pairs. Arbitrary injection is not
+    allowed. The parameterized job must allow the key to be injected.
```

Comment:
Can we document what the behavior is for a default value? Does this overwrite the existing value?